test/conftest.py

```diff
@@ -31,7 +32,10 @@
 LOOKUP_PED = os.path.join(INPUT, 'mock_sm_lookup.json')
 
 # can't use a session-scoped monkeypatch, so just set
-os.putenv('CPG_CONFIG_PATH', os.path.join(INPUT, 'cpg_runtime_config.toml'))
+os.putenv(
```

Comment:
`config-template.toml` loaded automatically in cpg-utils, so you don't need to set it explicitly!